src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -85,6 +112,9 @@ typedef bmpl::vector<
     current<UsedParticleCurrentSolver>,
     massRatio<MassRatioElectrons>,
     chargeRatio<ChargeRatioElectrons>
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
```

Comment:
*offline discussion with @Heikman and @Flamefire:*
Please check if the list
```c++
typedef typename PMacc::particles::traits::FilterByFlag<VectorAllSpecies,
+                                                                synchrotronPhotons<> >::type AllSynchrotronPhotonsSpecies;
```
is empty or not. If it is not empty, add photons.